_data/languages.yml

```diff
@@ -584,6 +775,19 @@
     morphosyntax: [ inflected ]
   territories: [ mg ]
 
+-
+  codes: [ mhr ]
+  names: [ Meadow Mari, Meadow-Eastern Mari, Eastern Mari ]
```

Comment:
I think if we have both "Meadow Mari" and "Eastern Mari", it's enough, people searching will find the language.